content/developer/reference/user_interface/view_architectures.rst

```diff
@@ -37,6 +37,26 @@ expression** that will be executed in an environment that has access to the foll
 - `uid (int)`: the id of the current user;
 - `today (str)`: the current local date in the `YYYY-MM-DD` format;
 - `now (str)`: the current local datetime in the `YYYY-MM-DD hh:mm:ss` format.
+- `companies (dict)`: the information of the companies. The dict contains:
```

Comment:
and here (we could describe the context here as being "all fields in the view" + parent (sometimes) + the basic eval context (link to where it is defined)